opsdroid/database/mongo/__init__.py

```diff
@@ -77,9 +86,12 @@ async def get(self, key):
 
         """
         _LOGGER.debug("Getting %s from MongoDB.", key)
-        return await self.database[key].find_one(
-            {"$query": {}, "$orderby": {"$natural": -1}}
+        response = await self.database[self.collection].find_one(
+            {"$query": {"_id": key}, "$orderby": {"$natural": -1}}
         )
+        if response.keys() == {"_id", "value"}:
+            response = response["value"]
```

Comment:
Did this data manipulation on retrieval in order to make the [memory example](https://docs.opsdroid.dev/en/latest/skills/memory.html#example) work without manipulation/adjustment.